.github/workflows/molecule.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
             # ensure we wipe out any data potentially still on the runner from previous run
             molecule reset
             # NOTE: for now we don't run "molecule test", because we don't have proper idempotence yet
-            molecule -v test --scenario-name default
+            molecule -v test --scenario-name user_provided
```

Comment:
(This might be just a leftover from testing this scenario, but in general we don't want to replace the default scenario but run both of them; we could even do that in separate jobs by using a [matrix](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#using-a-matrix-strategy))